client/state/wp-data/calypso-wp-data-store.js

```diff
@@ -0,0 +1,20 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { fetchPreferences, setPreference, savePreference } from 'state/preferences/actions';
+import { getPreference, isFetchingPreferences } from 'state/preferences/selectors';
+
+export default {
+	useCalypsoStore: true,
+	selectors: {
+		getPreference,
+		isFetchingPreferences,
+	},
+	actions: {
+		fetchPreferences,
+		setPreference,
+		savePreference,
+	},
+};
```

Comment:
I was also expecting to see a reducer here. Do you think it would make sense to migrate part of the existing reducer to wp.data as well? It would definitely contribute to having a better picture of how a complete example would look.

Reply:
Making reducers explicit here might help untangle the state tree (relevant for code-splitting?). I guess we'd need to extend the adapter to enable us to attach its collected top-level reducers to Calypso's state tree?

Reply:
I agree, but we'll need to add dynamic reducer functionality to the Calypso store as a pre-requisite. I'll start on another PR for that.

Reply:
> I was also expecting to see a reducer here.

I think it helps to look at it through the "interface vs implementation" lens. The module exports something that adheres to store interface, i.e. it has selectors and action. Reducer is a part of implementation. It's possible to have valid implementations that don't have any reducer at all.